test/list_parameter_test.py

```diff
@@ -81,7 +81,7 @@ def test_schema(self):
             a.normalize(["INVALID_ATTRIBUTE"])
 
         # Check that empty list is not valid
-        with pytest.raises(ValidationError, match=r"\[\] is too short"):
+        with pytest.raises(ValidationError):
```

Comment:
CI failed because jsonschema validation error is changed.

I think the message should not be compared because it will be very flaky test. Do you think about this?

```
# before
with pytest.raises(ValidationError, match=r"\[\] is too short"):
    a.normalize([])

# after
with pytest.raises(ValidationError, '[] should be non-empty'):
    a.normalize([])
```